selene-lib/default_std/lua51.toml

```diff
@@ -292,16 +292,16 @@ type = "number"
 [[debug.setupvalue.args]]
 type = "string"
 
-[[debug.traceback.args]]
-type = "function"
+[[debug.traceback.args]] # Since all of the arguments are optional and disregard order, the best thing to do is make them untyped.

```

Comment:
```suggestion
# Since all of the arguments are optional and disregard order, the best thing to do is make them untyped.
[[debug.traceback.args]]
```